cms/static/sass/elements/_modal-window.scss

```diff
@@ -777,4 +780,43 @@
       }
     }
   }
+
+  .modal-section-content {
+    @include font-size(16);
+
+    .group-select-title {
```

Comment:
I think the CSS that is specific to the new access editor should be scoped within a class so that it does not accidentally impact other editors. Perhaps put within the edit-unit-access class?